CGMBLEKit.xcodeproj/project.pbxproj

```diff
@@ -1111,6 +1113,50 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${SRCROOT}/Scripts/copy-frameworks.sh\"\n";
 		};
+		A906DD9D23DFAEAB000B74DC /* Copy Frameworks with Carthage */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"$(BUILT_PRODUCTS_DIR)/ShareClient.framework",
+				"$(BUILT_PRODUCTS_DIR)/ShareClientUI.framework",
+				"$(BUILT_PRODUCTS_DIR)/CGMBLEKit.framework",
+				"$(BUILT_PRODUCTS_DIR)/CGMBLEKitUI.framework",
+			);
```

Comment:
Recently, I needed to change `BUILT_PRODUCTS_DIR` -> `TARGET_BUILD_DIR` to avoid a `invalid symlink (absolute symlink forbidden)` error. Is this required here?

Reply:
Looks like the copy-frameworks.sh included is the updated one that resolves symlinks, so this should be fine.